controllers/workflow_controller.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2022 Hewlett Packard Enterprise Development LP.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"strings"
+
+	dwsv1alpha1 "github.com/HewlettPackard/dws/api/v1alpha1"
+	dwdparse "github.com/HewlettPackard/dws/utils/dwdparse"
+	"github.com/HewlettPackard/dws/utils/updater"
+	"github.com/go-logr/logr"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const DRIVERID string = "tester"
+
+// WorkflowReconciler reconciles a Workflow object
+type WorkflowReconciler struct {
+	client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=dws.cray.hpe.com,resources=workflows,verbs=get;list;watch;update;patch
+//+kubebuilder:rbac:groups=dws.cray.hpe.com,resources=workflows/status,verbs=get;update;patch
```

Comment:
It looks like we don't need this line, since the status is not a subresource in this case.
See https://github.com/NearNodeFlash/nnf-sos/blob/master/controllers/nnf_workflow_controller.go#L72

Then 'make manifests' so it updates config/rbac again.